src/transformers/pipelines/audio_utils.py

```diff
@@ -123,9 +146,13 @@ def ffmpeg_microphone_live(
             The length of the striding to be used. Stride is used to provide context to a model on the (left, right) of
             an audio sample but without using that part to actually make the prediction. Setting this does not change
             the length of the chunk.
-        format_for_conversion (`str`, defalts to `f32le`)
+        format_for_conversion (`str`, defaults to `f32le`)
```

Comment:
nit - in terms of the docstrings, we use "optional" to denote if it's a required argument, or the user can optionally specify it 

```suggestion
        format_for_conversion (`str`, *optional*, defaults to `f32le`)
```